content/authors/hpatro.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Harkrishn Patro
+extra:
+    photo: '/assets/media/authors/hpatro.png'
```

Comment:
I prefer a real photo, but if you don't have one you like, no problem.

Reply:
Couldn't decide which photo to add and copied a default one. :D